.github/workflows/linter.yml

```diff
@@ -1,4 +1,4 @@
-name: Lint
+name: Linter
```

Comment:
Renamed to disambiguate this workflow from the other workflow named `Lint`.

Before | After
--- | ---
<img width="271" alt="before" src="https://user-images.githubusercontent.com/3699047/159378884-d9c78051-755c-4911-bcc8-e40801844137.png"> | <img width="277" alt="after" src="https://user-images.githubusercontent.com/3699047/159378895-c85c71c3-cc8f-4201-9f04-7e68685a41be.png">